### DISCUSSION TITLE
Add filtering by errors

### DISCUSSION THREAD

**Discussion by ankerbachryhl** [langfuse/langfuse#1287] [Ideas]
### Describe the feature or potential improvement

More flexible filters for generations. Especially filtering by errors.

**Comment by marcklingen** [comment]
thx, will do

cc @RichardKruemmel

**Comment by RichardKruemmel** [comment]
@ankerbachryhl
We introduced more filter options across our tables including the generations table. You can now also filter by errors utilizing the level or status message column.
